components/wrapper/Wrapper.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function Wrapper({ styles, children }) {
+  return (
+    <section className={`px-5 lg:px-16 md:px-9 mx-auto xl:max-w-8xl ${styles}`}>
+      {children}
+    </section>
+  );
+}
```

Comment:
Why is the Wrapper taking `styles` as props?  I think Yannick instructed to avoid passing CSS Styles as props besides if it being applied from the `_app.js` are we really going to use it elsewhere with different styles?